octobot/storage/trading_metadata.py

```diff
@@ -54,96 +56,147 @@ async def store_backtesting_run_metadata(exchange_managers, start_time, user_inp
 
 async def _get_trading_metadata(exchange_managers, run_start_time, user_inputs, run_dbs_identifier, is_backtesting) \
```

Comment:
Could we split this method to improves its readability please ?

Reply:
yes !